.circleci/config.yml

```
@@ -76,6 +76,14 @@ executors:
         resource_class: <<parameters.resource_class>>
 
 commands:
+    create_docker_context:
```

To be consistent with other commands:
```suggestion
    create-docker-context:
```